codecov.yml

```diff
@@ -8,9 +8,9 @@
 codecov:
   require_ci_to_pass: yes
   notify:
-    # We have at least 10 builds in GitHub Actions and 12 in Azure
-    # and lint + mypy + docs + ReadTheDocs
-    after_n_builds: 15
+    # We have at least 5 builds in GitHub Actions,
+    # so try not to send the reports too soon with partial results.
+    after_n_builds: 5
```

Comment:
... the joy of manually counting small numbers of things because the computer cannot manage to do it ...